### ISSUE TITLE
Getting an error opening the gallery on iOS

### ISSUE THREAD

**Issue by darkmantle** [manijak/nativescript-photoviewer#43]
When I open the gallery it loads fine and works, however it's a few seconds delay due to the following console error:

`CONSOLE ERROR file:///app/vendor.js:51628:24: ERROR Error: Uncaught (in promise): Error: This value is not a native object.`

My array of photos is ok, giving correct URL's (tested and confirmed).

```ts
this.endTimer();
        const photoOptions: PhotoViewerOptions = {
            startIndex: this.currentImage,
            ios: {
                completionCallback: this.galleryLoaded
            }
        }

        const photoView = new PhotoViewer();
        photoView.showGallery(this.photos, photoOptions).then(() => {
            console.log("gallery closed");
            this.startTimer();
        });
```

**Comment by manijak** [comment]
As I have stated in the readme, initialization must happen in the onloaded event
`photoView = new PhotoViewer();`

**Comment by darkmantle** [comment]
Ah ok, thank you.

**Comment by darkmantle** [comment]
Sorry, idk why I clicked close again! I tried doing that, but it still prints the error. Although it works, it's annoying as it will trigger a bug report when in production.

I'm using Angular if that makes a difference.

**Comment by manijak** [comment]
Ah yeah, the famous angular. :-) sorry have not confirmed that the plugin works with angular nor vue. Need time to iron that out

**Comment by manijak** [comment]
Published version 2.1.0 just now. Promise handling should work better for both iOS and Android on both vanilla TNS and Angular.

**Comment by darkmantle** [comment]
Awesome. Works great now!
